src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedDayNames.cs

```diff
@@ -23,11 +23,11 @@ public void AbbreviatedDayNames_Set()
             Assert.Equal(newAbbreviatedDayNames, format.AbbreviatedDayNames);
         }
 
-        [Fact]
+        [Fact ActiveIssue(13247, TestPlatforms.AnyUnix)]
         public void AbbreviatedDayNames_Set_Invalid()
         {
             Assert.Throws<ArgumentNullException>("value", () => new DateTimeFormatInfo().AbbreviatedDayNames = null); // Value is null
-            Assert.Throws<ArgumentNullException>("value", () => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
+            Assert.Throws<ArgumentNullException>("values[3]", () => new DateTimeFormatInfo().AbbreviatedDayNames = new string[] { "1", "2", "3", null, "5", "6", "7" }); // Value has null
```

Comment:
we need to revert  ActiveIssue(13247, TestPlatforms.AnyUnix) too not just the code. just double checking

---
In reply to: [86198590](https://github.com/dotnet/corefx/pull/13229#discussion_r86198590) [](ancestors = 86198590)